### PR TITLE
CY-247 Add hidden value secret documentation

### PR DESCRIPTION
**Pull request by inbala**

### CODE REVIEW

File: content/_secrets.md [hunk 1]
```diff
@@ -312,6 +320,8 @@ Updates a secret.
 Property | Type | Description
 --------- | ------- | -----------
 `value` | string | The secret's new value.
+`visibility` | string | Optional parameter, defines who can see the secret (default: tenant). **Supported for Cloudify Manager 4.3 and above.**
+`is_hidden_value` | boolean| Optional parameter, determines who can see the secret's value, if True the secret's value is only shown to the user that created the secret and to admins. (default: false). **Supported for Cloudify Manager 4.4 and above.**
```
Comment (bmansheim) [inline content/_secrets.md]: `is_hidden_value` | boolean| Optional parameter, determines who can see the value of the secret. If True, the value of the secret is only shown to the user that created the secret and to admins. (default: false). **Supported for Cloudify Manager 4.4 and above.**

File: content/_secrets.md [hunk 2]
```diff
@@ -233,6 +239,7 @@ Property | Type | Description
 `value` | string | The secret's value.
 `update_if_exists` | boolean | Update value if secret already exists (optional, defaults to false).
 `visibility` | string | Optional parameter, defines who can see the secret (default: tenant). **Supported for Cloudify Manager 4.3 and above.**
+`is_hidden_value` | boolean| Optional parameter, determines who can see the secret's value, if True the secret's value is only shown to the user that created the secret and to admins. (default: false). **Supported for Cloudify Manager 4.4 and above.**
```
Comment (bmansheim) [inline content/_secrets.md]: `is_hidden_value` | boolean| Optional parameter, determines who can see the value of the secret. If True, the value of the secret is only shown to the user that created the secret and to admins. (default: false). **Supported for Cloudify Manager 4.4 and above.**

File: content/_secrets.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ Attribute | Type | Description
 `updated_at` | datetime | The time the secret was last updated at.
 `value` | string | The secret's value.
 `visibility` | string | Defines who can see the secret. Can be private, tenant or global. **Supported for Cloudify Manager 4.3 and above.**
+`is_hidden_value` | boolean| Determines who can see the secret's value, if true the secret's value is only shown to the user that created the secret and to admins. (default: false). **Supported for Cloudify Manager 4.4 and above.**
```
Comment (bmansheim) [inline content/_secrets.md]: @inbala Is this not optional like the other two?
`is_hidden_value` | boolean| Determines who can see the value of the secret. If True, the value of the secret is only shown to the user that created the secret and to admins. (default: false). **Supported for Cloudify Manager 4.4 and above.**

Comment (inbala) [reply content/_secrets.md]: It is, but in this section of attributes we just explain the attributes, the other parts describe it as a parameter to an API call.

**Pull request merged**